src/hardwareio/IMUCalibDumper/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#################################################################################
+#                                                                               #
+# Copyright (C) 2019 Fondazione Istituto Italiano di Tecnologia (IIT)           #
+# All Rights Reserved.                                                          #
+#                                                                               #
+#################################################################################
+
+# @authors: Leandro de Souza Rosa <leandro.desouzarosa@iit.it>
+
+project(IMUDataDumper)
+
+file(GLOB source src/*.cpp)
```

Comment:
I prefer not to use GLOB to get src and header files when it's not really necessary. In this case it is very easy to specify the src dircely in the `add_executable` call.